Jint/DebuggerStatementHandling.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Jint
+{
+    public enum DebuggerStatementHandling
+    {
+        Ignore,
```

Comment:
as this is public, can we have short descriptions about modes?

Reply:
I've added descriptions for the enum members and fixed XML comments in general - replacing <code> with inline <c>. Also renamed the `Jint` member to `Script`.